doc/api/brotli.md

```diff
@@ -359,46 +423,69 @@ All of these take a [`Buffer`][], [`TypedArray`][], [`DataView`][],
 to supply options to the `brotli` classes and will call the supplied callback
 with `callback(error, result)`.
 
-Every method has a `*Sync` counterpart, which accept the same arguments, but
+Every method has a `*Sync` counterpart, which accepts the same arguments, but
 without a callback.
 
-### brotli.compress(buffer[, options][, callback])
+### brotli.compress(data[, options][, callback])
+<!-- YAML
+added: REPLACEME
+-->
 
-- `buffer` {Buffer|TypedArray|DataView|ArrayBuffer|string}
+- `data` {Buffer|TypedArray|DataView|ArrayBuffer|string}
+- `options` {Object} an [options][] object.
+- `callback` {Function}
 
-Decompress a chunk of data with [Decompress][].
-If callback is omitted a Promise will be returned.
+Compress a chunk of data.
+If callback is omitted a {Promise} will be returned.
 
-### brotli.compressSync(buffer[, options])
+### brotli.compressSync(data[, options])
+<!-- YAML
+added: REPLACEME
+-->
 
-- `buffer` {Buffer|TypedArray|DataView|ArrayBuffer|string}
+- `data` {Buffer|TypedArray|DataView|ArrayBuffer|string}
+- `options` {Object} an [options][] object.
 
-Compress a chunk of data with [Compress][].
+Compress a chunk of data.
 
-### brotli.decompress(buffer[, options][, callback])
+### brotli.decompress(data[, options][, callback])
+<!-- YAML
+added: REPLACEME
+-->
 
-- `buffer` {Buffer|TypedArray|DataView|ArrayBuffer|string}
+- `data` {Buffer|TypedArray|DataView|ArrayBuffer|string}
+- `options` {Object} an [options][] object.
 
-Decompress a chunk of data with [Decompress][].
-If callback is omitted a Promise will be returned.
+Decompress a chunk of data.
+If callback is omitted a {Promise} will be returned.
 
-### brotli.decompressSync(buffer[, options])
+### brotli.decompressSync(data[, options])
+<!-- YAML
+added: REPLACEME
+-->
 
-- `buffer` {Buffer|TypedArray|DataView|ArrayBuffer|string}
+- `data` {Buffer|TypedArray|DataView|ArrayBuffer|string}
+- `options` {Object} an [options][] object.
+- `callback` {Function}
 
-Decompress a chunk of data with [Decompress][].
+Decompress a chunk of data.
 
-[`.flush()`]: #brotli_brotli_flush_kind_callback
+[`.flush()`]: #brotli_brotli_flush_callback
+[`.setParameter()`]: #brotli_brotli_setparameter_param_value
 [`Accept-Encoding`]: https://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html#sec14.3
 [`ArrayBuffer`]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/ArrayBuffer
-[`brotli/encode.h`]: https://github.com/google/brotli/blob/v1.0.4/c/include/brotli/encode.h
+[`brotli.constants`]: #brotli_constants
```

Comment:
If this intended to refer `brotli.constants` property (and not jusst "Constatnts" section), then this shoud be `#brotli_brotli_constants`.